Frameworks/CoreText/CTLine.mm

```diff
@@ -240,7 +240,10 @@ void _CTLineDraw(CTLineRef lineRef, CGContextRef ctx, bool adjustTextPosition) {
             CGContextSetTextPosition(ctx, curTextPos.x + curRun->_relativeXOffset, curTextPos.y);
         }
 
-        _CTRunDraw(static_cast<CTRunRef>(curRun), ctx, CFRange{}, false);
+        // Get height of the line so we draw with the correct baseline for each run
+        CGFloat ascent, descent;
+        CTLineGetTypographicBounds(lineRef, &ascent, &descent, nullptr);
+        _CTRunDraw(static_cast<CTRunRef>(curRun), ctx, CFRange{}, false, ascent - descent);
```

Comment:
we discussed offline - my mental model of the baseline was wrong, we should just use ascent.

---
In reply to: [84551992](https://github.com/Microsoft/WinObjC/pull/1200#discussion_r84551992) [](ancestors = 84551992)